src/main/java/de/uni_stuttgart/it_rex/quiz/web/rest/written/QuestionResource.java

```diff
@@ -125,6 +127,19 @@ public QuestionResource(
         return questionService.findAll(courseId);
     }
 
+    /**
+     * {@code GET  /questions} : get all the questions by ids.
+     *
+     * @param questionIds Questions Ids as comma separated list.
+     * @return the map of questions in the body.
+     */
+    @GetMapping(value="/questions", params="question_ids")
+    public Map<UUID, QuestionDTO> findAllByIds(@RequestParam("question_ids") final List<UUID> questionIds) {
+        log.info("REST request to get all questions by ids: {}", questionIds);
+        final List<QuestionDTO> questions = questionService.findByIdIn(questionIds);
+        return questions.stream().collect(Collectors.toMap(QuestionDTO::getId, quiz -> quiz));
```

Comment:
Ich weiß nicht ob ichs richtig verstehe, aber sollten hier nicht die questions in der Map als entität zurückgegeben werden? Aktuell werden die Quizzes zurückgegeben.

Reply:
ah typo.. macht aber keinen unterschied :)